kubernetes-services/work-reaper/db/db.sql

```diff
@@ -0,0 +1,135 @@
+--- This file contains the commands to create a database from scratch
```

Comment:
We shouldn't have multiple copies of the database schema file - we should just update whatever scripts are reading this to use the main harmony repo one (I assume just the `create database` scripts?

Also there should ideally only be one create-database script which should take in the directory where we want it created.

Reply:
We can address this (along with code duplication) in https://bugs.earthdata.nasa.gov/browse/HARMONY-1538.